lib/utils.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
```

Comment:
Can you explain this `utils.js` file? Was this something you wrote or is it compiled javascript that you copied in from another source?

Reply:
This file was automatically generated at the `utils.ts` compilation. Since the former `verifyClaim` function was declared inside it and then imported in `LoginStrategy.ts` file, the corresponding .js file as been compiled when I made changes to it.

Reply:
Is it necessary to include in the project? Can you import from the `utils.ts` file?

Reply:
Since those .js files are just build on ts files, it is not necessary indeed to include in the project.